packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/feature_login/ui_components/login_form.dart

```diff
@@ -92,13 +94,12 @@ class _LoginFormState extends State<LoginForm> {
         },
       );
 
-  Widget _buildErrorListener() => RxBlocListener<LoginBlocType, String>(
+  Widget _buildErrorListener() => RxBlocListener<LoginBlocType, ErrorModel>(
```

Comment:
I guess the `RxBlocListener` can be added to the build method while the actual listener callback to be extracted as a private function